### PR TITLE
Make clearer to not use channel option and adding 403 error to troubleshooting.

### PR DESCRIPTION
**Pull request by Cristiandis**

### CODE REVIEW

File: src/Configuration/index.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ This section are the available options for the Studio and Player configurations,
 
 | Option             | Description                                                                                      | Default            |
 | ------------------ | ------------------------------------------------------------------------------------------------ | ------------------ |
-| `channel`          | the deployment channel to be used; **DO NOT CHANGE, ONLY KEPT FOR DEVELOPERS**                   | `"LIVE"`           |
+| ~~`channel`~~          | the deployment channel to be used; **DO NOT CHANGE, ONLY KEPT FOR DEVELOPERS**                   | `"LIVE"`           |
```
Comment (Noted-Jrelvas) [inline src/Configuration/index.md:41]: I feel like this change is unnecessary.

File: src/Troubleshooting/index.md [hunk 2]
```diff
@@ -77,6 +77,10 @@ There's no known fix (as of October 14th, 2023), but the following workarounds m
   If none of the workarounds above work or apply, consider using Roblox windowed instead of fullscreen.
   This may also be your GPU not supporting Vulkan or not supporting modern Vulkan; Use the OpenGL renderer, or set the installed DXVK version to one which includes a legacy version of Vulkan that your GPU supports.
 
+### 403: Forbidden while trying to download Studio/Player
+
+Due to Roblox locking down all channels except `live` if there's any value in the `channel` option Roblox will throw a 403 error, this can be easily fixed by removing the `channel` option.
+
```
Comment (Noted-Jrelvas) [inline src/Troubleshooting/index.md:83]: I dislike the way this is worded. Feels like a word salad.

**Review by Noted-Jrelvas — CHANGES_REQUESTED** [review]

File: src/Troubleshooting/index.md [hunk 3]
```diff
@@ -79,7 +79,7 @@ There's no known fix (as of October 14th, 2023), but the following workarounds m
 
 ### 403: Forbidden while trying to download Studio/Player
 
-Due to Roblox locking down all channels except `live` if there's any value in the `channel` option Roblox will throw a 403 error, this can be easily fixed by removing the `channel` option.
+This is caused by Roblox locking all channels except `live`, this can be easily fixed by removing the `channel` option.
```
Comment (Noted-Jrelvas) [inline src/Troubleshooting/index.md:82]:
```suggestion
On Oct. 2023, [Roblox decided to restrict access to all channels](https://devforum.roblox.com/t/-/2487241/7). If the `channel` option is set to anything other than `live`, you will run into this issue. This option should be removed from your configuration.
```
Hmm...

**Comment by lunarlettuce** [comment]
That's not true, having the channel selector set to anything at all will cause a 403.

**Comment by Noted-Jrelvas** [comment]
Excellent. Less words needed in docs then.

File: src/Troubleshooting/index.md [hunk 4]
```diff
@@ -77,6 +77,10 @@ There's no known fix (as of October 14th, 2023), but the following workarounds m
   If none of the workarounds above work or apply, consider using Roblox windowed instead of fullscreen.
   This may also be your GPU not supporting Vulkan or not supporting modern Vulkan; Use the OpenGL renderer, or set the installed DXVK version to one which includes a legacy version of Vulkan that your GPU supports.
 
+### 403: Forbidden while trying to download Studio/Player
+
+This is caused by Roblox locking all channels except `live`, this can be easily fixed by removing the `channel` option.
```
Comment (Noted-Jrelvas) [inline src/Troubleshooting/index.md:82]:
```suggestion
On Oct. 2023, [Roblox decided to restrict access to all channels](https://devforum.roblox.com/t/-/2487241/7). If the `channel` option is set, you will run into this issue. It should be removed from your configuration.
```

**Pull request closed without merging**